packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices/Choices.stories.js

```diff
@@ -0,0 +1,72 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import React from 'react'
+
+import Choices from './Choices'
+import task from '../../../../../data/task'
+
+const filteredChoices = Array.from(task.choicesOrder)
+
+function StoryContext (props) {
+  const { children, theme } = props
+
+  return (
+    <Grommet
+      background={{
+        dark: 'dark-1',
+        light: 'light-1'
+      }}
+      theme={theme}
+      themeMode={(theme.dark) ? 'dark' : 'light'}
+    >
+      <Box
+        background={{
+          dark: 'dark-3',
+          light: 'neutral-6'
+        }}
+        pad='1em'
+        width='380px'
+      >
+        {children}
+      </Box>
+    </Grommet>
+  )
+}
+
+export default {
+  title: 'Tasks / SurveyTask / Choices',
+  component: Choices
+}
+
+const Template = ({ dark, filteredChoices, task }) => (
+  <StoryContext
+    theme={{ ...zooTheme, dark }}
+  >
+    <Choices
+      filteredChoices={filteredChoices}
+      onChoose={() => console.log('button clicked')}
+      task={task}
+    />
+  </StoryContext>
+)
+
```

Comment:
Should add a story for more than 30 choices (no thumbnails), but the mock survey task currently using (from staging workflow, built from project builder example CSVs) has less than 30 choices, and getting to greater than 30 choices will take some work, so would like to postpone for now.

```suggestion
// add MoreThanThirty story with refactored mock survey task
```